openedx/openedx-platform
# refactor: remove ContentObjectTag model and related functions

## Description

This PR removes the ContentObjectTag model and some overridden tagging functions from `oel_tagging.`

## Testing instruction
Please ensure that the tests cover the expected behavior of the refactor.

___
Private-ref: [FAL-3610](https://tasks.opencraft.com/browse/FAL-3610)
